code/modules/mob/living/carbon/human/human_defines_vr.dm

```diff
@@ -7,3 +7,4 @@
 	var/flapping = 0
 	var/vantag_pref = VANTAG_NONE //What's my status?
 	var/impersonate_bodytype //For impersonating a bodytype
+	var/teeth = 32 //How many teeth are left in your mouth
```

Comment:
Unacceptable, not all species on the station have 32 teeth. Some may not have teeth at all, some might have chitinous mandibles or somesuch that superficially resemble the function of teeth but are not themselves teeth

Reply:
For the species, which do not posses teeth, I have added a variable in species defines.
As for my mistake of assuming that all species have equal amount of teeth - I admit that oversight. Where, or from whom can I get the exact numbers. I do wish to correct my mistake so.